_posts/2017-03-31-introducing-deckgl-v4.md

```diff
@@ -155,14 +144,14 @@ We have already started experimenting with the powerful features in the newly re
 
 In the new v4 release of deck.gl, some new layers like the `GeoJsonLayer` and `HexagonLayer` are what we called **composite** layers as they are actually built by "composing" other core deck.gl layers. The composite layers have their own props but leverage its underlying layers for part of their functionalities. This kind of composition greatly reduces the effort required for creating new layers and makes the internal structure of many layers much cleaner. In future releases, we will continue making composite layer development easier by decoupling different sets of functionalities of layers and keeping improving our guides and docs.
 
-### More Advanced Picking (Selection) Support
+### More Advanced Picking Support
 
-Picking or selection is also one of the most common tasks of any data visualization applications and is a core functionality we provide with deck.gl. In the v4 release, a color-coded picking process that are both much more robust and performant is implemented. We plan to continue improving picking by adopting better picking boundary arbitration, supporting square or polygon area picking, etc.
+Picking (e.g. using a mouse to select the graphical representation of one data point to highlight it or show more information about it) is one of the most common tasks of any data visualization applications and is a core functionality we provide with deck.gl. We have ideas for improving picking by adopting better picking boundary arbitration (bigger pick targets), supporting square or polygon area picking, etc.
```

Comment:
I believe "Mouse picking or mouse selection" is a self-explanatory. But if you think it's better to be more verbose. I'm okay with it.